hazelcast/src/main/java/com/hazelcast/ringbuffer/impl/RingbufferContainer.java

```diff
@@ -43,12 +49,11 @@
  * it is null to save space.
  */
 @SuppressWarnings("checkstyle:methodcount")
-public class RingbufferContainer implements IdentifiedDataSerializable {
+public class RingbufferContainer<T> implements IdentifiedDataSerializable, Notifier, Versioned {
```

Comment:
Ah, this could probably be switched to the event journal PR.

Yes, until now only operations could notify other parked operations. Now, all sorts of map/cache and scheduled tasks such as eviction/expiration can notify parked operations since they add events to the event journal. Since the operation unpark method requires an implementation of  `Notifier` I have to provide one every time an event is added to the journal. Making the container a notifier avoids allocation of a new anonymous implementation.